lib/widgets/shipping_address_input_form.dart

```diff
@@ -172,32 +184,44 @@ class _ShippingAddressInputFormState extends State<ShippingAddressInputForm> {
       return;
     }
 
-    try {
-      final http.Response response = await http.get(
-        'https://us-central1-badiup2.cloudfunctions.net/api/postcodes/' +
-            widget.postcodeTextController.text,
-        headers: {
-          "content-type": "application/json",
-          "accept": "application/json",
-        },
-      );
-
-      String responseString =
-          response.body.replaceAll('\\n', '').replaceAll('\\', '');
-      responseString = responseString.substring(1, responseString.length - 1);
-      Map<String, dynamic> addressInfo = jsonDecode(responseString);
-      Map<String, dynamic> addressInfoInternal =
-          (addressInfo['data'] as List).first;
+    setState(() {
+      widget.addressSearchStatusController.inProgress = true;
```

Comment:
@dragonfly91 Please educate me...:pray: So this is the equivalent of calling setter `InProgress`?
```
class StatusController extends ValueNotifier<bool> {
  StatusController({bool value = false}) : super(value);

  bool get inProgress => value;

  set inProgress(bool newValue) {
    value = newValue;
  }
}
```

Reply:
Yes! :) More info: https://stackoverflow.com/questions/27683924/how-do-getters-and-setters-change-properties-in-dart

Reply:
@dragonfly91 Thank you for you comment! Wow~ :)